lms/djangoapps/commerce/api/v0/tests/test_views.py

```diff
@@ -160,12 +166,12 @@ def test_ecommerce_api_error(self):
         self.assertValidEcommerceInternalRequestErrorResponse(response)
         self.assertUserNotEnrolled()
 
-    def _test_successful_ecommerce_api_call(self, is_completed=True):
+    def _test_successful_ecommerce_api_call(self, is_completed=True, utm_tracking_present=False):
```

Comment:
Now that this takes 2 `kwargs` can we update the places in the code that are just calling `_test_successful_ecommerce_api_call(False)` to be `_test_successful_ecommerce_api_call(is_completed=False)` so there is no ambiguity